### DISCUSSION TITLE
Run Frigate stand-alone (without home assistant) ?

### DISCUSSION THREAD

**Discussion by rubenwylleman** [blakeblackshear/frigate#2658] [Ask A Question] [answered]
Hi,

I'm quite impressed by the software. 
As far as i understand, this only runs in the Home assistant environment? (correct me if i'm wrong)
Is there any intention to make this software run stand alone, with it's own GUI etc ?

Anyway, keep up the good work.

**Comment by blakeblackshear** [comment, marked as answer]
It runs stand-alone and has it's own web interface. It does require an mqtt server, but not home assistant. It also does not provide any kind of built-in authentication.

**Reply by rubenwylleman** [reply]
Thanks for the quick reply !!

I'll try setting it up in the near future.
